game/models.py

```diff
@@ -12,6 +12,9 @@ class Reward(models.Model):
     """Representation of the reward of the game"""
     description = models.TextField()
 
+    def __str__(self):
+        return self.description[:16]
```

Comment:
Maybe long and short description? Long to get details, short to represent Reward